### PR TITLE
Update Competition Templates and add Polish translation

### PR DESCRIPTION
**Pull request by dmint789**
This also updates the competition rules image in the competitor tutorial for all languages. All changes are minor cosmetic tweaks, none of the wording has been changed.

### CODE REVIEW

**Review by DanielEgdal — DISMISSED** [review]
Approving from Nevins

**Review by Nanush7 — APPROVED** [review]
LGTM!

**Comment by dmint789** [comment]
For the record, the English version was updated considerably, so all other translations must be updated now

File: edudoc/competition-templates/competition-templates.md [hunk 1]
```diff
@@ -4,59 +4,66 @@
...
 
-1. If you would like to use the WCA designs, use the files in the folder for the language used in your region. **WARNING: Some of the PDF previews might have rendering artifacts, but these won't show up if you download the files.** If you would like to make custom designs, use the files in the `Editable Files` folder. **DO NOT** make colored Clock cover designs, they must be in black and white (BW).
-2. The designs for paper Clock covers and competition rules can be printed on a home printer; the Clock covers need to be printed in BW and the competition rules need to be printed in color. The other designs **MUST** be printed using an industry-grade printer, so you will have to find a printing place in your area. They will require `PDF` files to make your prints.
+1. If you would like to use the WCA designs, use the files in the folder for the language used in your region, if available. If you would like to make custom designs, use the files in the `Editable Files` folder. **DO NOT** make Clock cover designs that use a lot of ink, because that makes the paper more rigid, which can affect Clock pins. It is preferable to have a minimal design in Black and White (BW).
```
Comment (Nanush7) [inline edudoc/competition-templates/competition-templates.md:21]:
```suggestion
1. If you would like to use the WCA designs, use the files in the folder for the language used in your region, if available. If you would like to make custom designs, use the files in the `Editable Files` folder.
```
This change would make sense if the Clock pins proposal is approved by the Board.

Comment (dmint789) [reply edudoc/competition-templates/competition-templates.md:21]: We should do that in a separate PR when that happens

**Review by nsilvestri — APPROVED** [review]
LGTM

**Pull request merged**